src/lib/dom.generated.d.ts

```diff
@@ -19225,7 +19232,7 @@ interface BlobCallback {
 }
 
 interface CustomElementConstructor {
-    new (): HTMLElement;
+    new (...params:[]): HTMLElement;
```

Comment:
Please add a space prior to the `[]`.

Uh, is this really supposed to be an empty tuple? What's the difference here?

Reply:
Oh yeah, that's wrong 👍